std/ndn/mgmt_2022/definitions.go

```diff
@@ -337,3 +338,19 @@ type CsQuery struct {
 	PacketSize      uint64
 	FreshnessPeriod uint64
 }
+
+type PrefixInjection struct {
```

Comment:
`mgmt_2022` is for NFD management right? are we treating this as part of the NFD management protocol?

Reply:
Ah I see what you mean, I'll move it out of the NFD management protocol defs for now